src/api/ApiConfig.ts

```diff
@@ -19,4 +19,34 @@ export default interface ApiConfig {
    * possible.
    * */
   refreshToken?: string;
+
+  /**
+   * Use cookie base auth instead of tokens - defaults to false
+   * */
+  cookieAuth?: boolean;
+}
+
+export interface DefaultApiConfig {
```

Comment:
As an interface, there's nothing really "default" about it - so this should just be `ApiConfig` right?

Reply:
I added it to have the correct type where apiUrl and cookieAuth are defined when applying defaults, do you have a more elegant solution?